cms_delete_content/static/src/js/delete_content.js

```diff
@@ -0,0 +1,58 @@
+odoo.define('cms_delete_content.delete_confirm', function (require) {
+    'use strict';
+
+    var ajax = require('web.ajax');
+    var msg_tool = require('cms_status_message.tool');
+    var core = require('web.core');
+    var _t = core._t;
+
+    $(document).ready(function () {
+      $(".cms_delete_confirm").click(function(evt){
+        evt.preventDefault();
+        // load modal
+        $.ajax({
+          method: "GET",
+          url: $(this).attr('href'),
+          data: {}
+        }).done(function (modal) {
+          var $modal = $(modal),
+              $form = $('form', $modal);
+          // handle modal form submit
+          $form.submit(function(evt){
```

Comment:
Please avoid using too many lambdas. When you switch to animation format as said above, then better make use of proxied methods inside the class itself, so that they can be also inherited/extended (besides more maintainable). Check how it's done in [the above example again](https://github.com/OCA/website/blob/9.0/website_snippet_country_dropdown/static/src/js/website_snippet_country_dropdown.js#L20).